### PR TITLE
34 - fetch languages from backend

### PR DESCRIPTION
**Pull request by stjepanbonic**
- fix cors for localhost
- some changes from IDE warnings

### CODE REVIEW

File: backend/src/main/java/com/triolingo/controller/TeacherController.java [hunk 1]
```diff
@@ -43,18 +43,18 @@ public TeacherController(TeacherService teacherService, TeacherTranslator teache
     }
 
     @GetMapping("/all")
-    @Secured("ROLE_ADMIN")
+    //@Secured("ROLE_GUEST")
```
Comment (stjepanbonic) [inline backend/src/main/java/com/triolingo/controller/TeacherController.java:46]: for table of teachers, can be accessed by anyone

File: backend/src/main/java/com/triolingo/controller/StudentController.java [hunk 2]
```diff
@@ -41,15 +41,15 @@ public StudentController(StudentService studentService, StudentTranslator studen
...
-        return studentService.listAll().stream().map(student -> studentTranslator.toDTO(student)).toList();
+        return studentService.listAll().stream().map(studentTranslator::toDTO).toList();
     }
 
     @GetMapping("/{id}")
     @Secured("ROLE_ADMIN")
     @Operation(description = "Returns information regarding student with {id}.")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200"),
-            @ApiResponse(responseCode = "404", content = @Content(schema = @Schema(implementation = Void.class)))
```
Comment (LeoTheMoldyLemon) [inline backend/src/main/java/com/triolingo/controller/StudentController.java:52]: Jesi li provjerio da swagger i dalje stavlja dobar return content ovako?

Comment (stjepanbonic) [reply backend/src/main/java/com/triolingo/controller/StudentController.java:52]: yup, radi

**Review by LeoTheMoldyLemon — APPROVED** [review]
LGTM

**Pull request merged**